### PR TITLE
updating terrascan results for version 1.6.0

### PR DESCRIPTION
**Pull request by harkirat22**
update terrascan results for latest version 1.6.0

### CODE REVIEW

File: test-cases/terraform/aws/encryption/at-rest/docdb_cluster_encrypted_at_rest_using_cmk_not_customer_managed/results_summary.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "cloudrail": "yes",
   "kics": "no",
   "snyk": "no",
-  "terrascan": "no",
+  "terrascan": "yes",
```
Comment (yi2020) [inline test-cases/terraform/aws/encryption/at-rest/docdb_cluster_encrypted_at_rest_using_cmk_not_customer_managed/results_summary.json:6]: In the output below I can't see DocDB cluster encryption violation

**Review by yi2020 — CHANGES_REQUESTED** [review]
Great work! Out of 57 updated cases, only one needs a review/update from you.

File: test-cases/terraform/aws/encryption/at-rest/docdb_cluster_encrypted_at_rest_using_cmk_not_customer_managed/results_summary.json (hunk 1 above)
Comment (harkirat22) [reply test-cases/terraform/aws/encryption/at-rest/docdb_cluster_encrypted_at_rest_using_cmk_not_customer_managed/results_summary.json:6]: @yi2020 sure, I will update this.

Comment (harkirat22) [reply test-cases/terraform/aws/encryption/at-rest/docdb_cluster_encrypted_at_rest_using_cmk_not_customer_managed/results_summary.json]: done, made the changes

**Pull request merged**